core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/utilities/JsonSerializer.java

```diff
@@ -34,7 +34,16 @@
 
     private final Logger logger = LoggerFactory.getLogger(JsonSerializer.class);
 
-    JsonObject writeTextAnnotation(TextAnnotation ta) {
+    public static final String FORM = "form";
+    public static final String STARTCHAROFFSET = "startCharOffset";
+    public static final String ENDCHAROFFSET = "endCharOffset";
+    public static final String TOKENOFFSETS = "tokenOffsets";
+
+    JsonObject writeTextAnnotation(TextAnnotation ta ) {
+        return writeTextAnnotation(ta, false);
+    }
+
+    JsonObject writeTextAnnotation(TextAnnotation ta, boolean doWriteTokenOffsets ) {
```

Comment:
Could you add tests which cover `doWriteTokenOffsets = true`? 
https://github.com/mssammon/illinois-cogcomp-nlp-1/blob/03e49ed5aa539445579462c16410a1e2fd12ac7f/core-utilities/src/test/java/edu/illinois/cs/cogcomp/annotation/TextAnnotationSerializationTest.java